pkg/scaffold/manager/cmd.go

```diff
@@ -57,6 +57,8 @@ import (
 )
 
 func main() {
+	var metricsBindAddr string
+	flag.StringVar(&metricsBindAddr, "metrics-bind-addr", ":8080", "The address the metric endpoint binds to.")
```

Comment:
nit: can we call the flag `metrics-addr` ? (internally we can use metricsBindAddr)